server/Dockerfile

```diff
@@ -1,5 +1,21 @@
+# This stage is tagged for local development.
+FROM node:16-slim AS dev
+WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y wget libcurl4
+RUN wget -O /usr/local/bin/liftOver https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver  \
+  && chmod +x /usr/local/bin/liftOver
+COPY . .
+WORKDIR /home/node
+RUN mkdir chains && \
+  wget -O ./chains/hg19ToHg38.over.chain.gz https://hgdownload.cse.ucsc.edu/goldenpath/hg19/liftOver/hg19ToHg38.over.chain.gz && \
+  wget -O ./chains/hg38ToHg19.over.chain.gz https://hgdownload.cse.ucsc.edu/goldenpath/hg38/liftOver/hg38ToHg19.over.chain.gz && \
+  gunzip ./chains/hg38ToHg19.over.chain.gz && \
+  gunzip ./chains/hg19ToHg38.over.chain.gz
+RUN chown -R node:node ./ && chown -R node:node /home/node
+WORKDIR /usr/src/app
```

Comment:
This isn't great as a base build stage. I think you can have something like this, but use a `COPY` directive later on to grab the files that you need. For example, a `download` stage using `curlimages/curl`, download and unzip, and then later on in a different stage, do `COPY --from=download --chown node:node STUFF DESTINATION`